src/lib/serviceRoutines/entryPointsTreat.h

```diff
@@ -0,0 +1,52 @@
+#ifndef SRC_LIB_SERVICEROUTINES_ENTRYPOINTS_TREAT_H_
+#define SRC_LIB_SERVICEROUTINES_ENTRYPOINTS_TREAT_H_
+
+/*
+*
+* Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: José Manuel Cantera
+*/
+#include <string>
+#include <vector>
+
+#include "rest/ConnectionInfo.h"
+#include "ngsi/ParseData.h"
+
+
+#define TYPES_URL            "/v2/types"
+#define ENTITIES_URL         "/v2/entities"
+#define SUBSCRIPTIONS_URL    "/v2/subscriptions"
+#define REGISTRATIONS_URL    "/v2/registrations"
+
```

Comment:
Are these defines used outside entryPointsTreat.cpp?
If not, put them in entryPointsTreat.cpp instead.